phys2bids/viz.py

```diff
@@ -66,7 +66,7 @@ def ntr2time(x):
     subplot.set_ylabel('Volts')
     subplot.plot(time, trigger, '-', time, thrline, 'r-.', time, block, '-')
     subplot.fill_between(time, block, where=block >= d, interpolate=True, color='#ffbb6e')
-    subplot.legend(["trigger", "input threshold", "time block"])
+    subplot.legend(["trigger", "input threshold", "time block"], loc='upper right')
```

Comment:
Why are you changing this?

Reply:
By default, `matplotlib.legend` defaults to `loc='best'`, where it attempts to place your legend in the best location based on the plotted data. When you have a _lot_ of data (as we often do in these plots) `matplotlib` will [raise a warning](https://github.com/matplotlib/matplotlib/blob/537a8c905e82fc24cb6d83042b30787aa396cd27/lib/matplotlib/legend.py#L1029-L1032) that using `loc='best'` is slower than manually specifying the location. Since (I think&mdash;but please correct me if I'm wrong!) the legend is supposed to always be in the upper right corner of the plot, setting the default location here should speed things up a bit.

Reply:
Let me know if you'd prefer that go in a separate PR! I was just doing it here because I was getting rid of all the warnings that the codebase was throwing in tests (e.g., I'm filtering out the `bioread` warning, as well).

Reply:
It's fine, just asking